lib/archethic/p2p/message/get_current_replications_attestations.ex

```diff
@@ -0,0 +1,95 @@
+defmodule Archethic.P2P.Message.GetCurrentReplicationsAttestations do
+  @moduledoc """
+  Represents a message to get the current beacon slots for a subset
+  """
+
+  @enforce_keys [:subsets]
+  defstruct [:subsets, :paging_address]
+
+  alias Archethic.Crypto
+  alias Archethic.BeaconChain.Slot
+  alias Archethic.BeaconChain.Subset
+  alias Archethic.P2P.Message.GetCurrentReplicationsAttestationsResponse
+  alias Archethic.Utils
+
+  @paging_size 100
+  @type t :: %__MODULE__{
+          subsets: list(binary()),
+          paging_address: nil | Crypto.prepended_hash()
+        }
+
+  @spec process(__MODULE__.t(), Crypto.key()) :: GetCurrentReplicationsAttestationsResponse.t()
+  def process(%__MODULE__{subsets: subsets, paging_address: paging_address}, _) do
+    replications_attestations =
```

Comment:
Here you should also retrieve the replication attestations from the SummaryCache.